feature/gnoi/os/tests/osinstall/osinstall_test.go

```diff
@@ -425,6 +425,10 @@ func watchStatus(t *testing.T, ic ospb.OS_InstallClient, standby bool) error {
 func TestPushAndVerifyInterfaceConfig(t *testing.T) {
 
 	dut := ondatra.DUT(t, "dut")
+	ocPortName := dut.Port(t, "port1").Name()
```

Comment:
`ocPortName` usage is gated by `deviations.ExplicitInterfaceInDefaultVRF(dut)`, therefore it should be included in the `if` statement.

Reply:
You're absolutely correct; this is fixed in latest commit.